src/components/Table/Table.tsx

```diff
@@ -5,7 +5,16 @@ import { testIds } from "./testIds";
 
 import table from "./Table.module.css";
 
-export const Table = ({ operationList }) => {
+ type ItemType = {
```

Comment:
It is a common thing to start types and interfaces with `I`(for interface)
```suggestion
 type IExpance = {
```